### PR TITLE
Created DTOs for SlowOperationDetector for ManagementCenter.

### PR DESCRIPTION
**Pull request by Donnerbart**
ManagementCenter was not able to deserialize the slow operation logs before. Created new DTO classes to provide this functionality.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalOperationStatsImpl.java [hunk 1]
```diff
@@ -84,19 +99,19 @@ public boolean equals(Object o) {
             return false;
         }
         LocalOperationStatsImpl that = (LocalOperationStatsImpl) o;
-        if (creationTime != that.creationTime) {
-            return false;
-        }
         if (maxVisibleSlowOperationCount != that.maxVisibleSlowOperationCount) {
             return false;
         }
         if (slowOperations != null ? !isEqualSlowOperations(that.slowOperations) : that.slowOperations != null) {
```
Comment (bilalyasar) [inline hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalOperationStatsImpl.java]: this is `if` very confusing :)

**Comment by Donnerbart** [comment]
verify

**Comment by pveentjer** [comment]
Can you rebase?

File: hazelcast/src/main/java/com/hazelcast/internal/management/dto/SlowOperationDTO.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import java.util.List;
+
+import static com.hazelcast.util.JsonUtil.getArray;
+import static com.hazelcast.util.JsonUtil.getInt;
+import static com.hazelcast.util.JsonUtil.getString;
+
+/**
+ * A Serializable DTO for {@link com.hazelcast.spi.impl.operationexecutor.slowoperationdetector.SlowOperationLog}.
+ */
+public class SlowOperationDTO {
```
Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/internal/management/dto/SlowOperationDTO.java]: The other DTO's are implementing JSON serializable. 

And for DTO's having getters and setters make no sense :) A DTO is not an object; it is a record.

Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/internal/management/dto/SlowOperationDTO.java]: Those DTOs are not (de)serialized by themselves. This is done exclusively via `LocalOperationStatsImpl`. So the `fromJSON()` method will never be used.

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/internal/management/dto/SlowOperationDTO.java]: Most (all?) of the DTO's in the managent.dto package implement JSonSerializable. So would make sure that your code is in line.

**Comment by Donnerbart** [comment]
Can you merge this, before you do the next refactoring @pveentjer?

**Comment by pveentjer** [comment]
Can you rebase? :)

**Pull request merged**